pylua/network.lua

```diff
@@ -0,0 +1,22 @@
+package.loadlib("libpython2.7.so", "*")
+require('lua-python')
+
+python.execute("import thread")
+python.execute("import time")
+python.execute("import lua")
+python.execute("from pylua.network import *")
+
+WebSocketClient = {}
+
+WebSocketClient.__index = WebSocketClient
+
+function WebSocketClient.WebSocketClient()
+	return python.eval("LuaWebSocketClient()")
```

Comment:
@OHerasym, are you following some style guide for lua code? On my opinion tabs are bad choice.